packages/next/src/lib/metadata/default-metadata.tsx

```diff
@@ -54,6 +54,6 @@ export function createDefaultMetadata(): ResolvedMetadata {
     bookmarks: null,
     category: null,
     classification: null,
-    other: {},
+    other: [],
```

Comment:
Hi,
Will the type change from `map` to `array` create problems for Next.JS developers' previous code?

Reply:
This change will not affect the previous functionality. I have included test cases for the "other" option, which cover both the previous and the new usage methods.